README.md

```diff
@@ -322,35 +319,29 @@ We encourage contributions from the community. No improvement is too small. Snap
 * _Contributors_: We can always use more eyes on the core framework and its testing
 * _Integrations_: Snap can collect from and publish to almost anything by [authoring a plugin](#author-a-plugin)
 
-To contribute to the Snap framework, see [our CONTRIBUTING file](CONTRIBUTING.md). To give back to a specific plugin, open an issue on its repository.
+To contribute to the Snap framework, see our [CONTRIBUTING.md](CONTRIBUTING.md) file. To give back to a specific plugin, open an issue on its repository. Snap maintainers aim to address comments and questions as quickly as possible. To get some attention on an issue, reach out to us [on Slack](http://slack.snap-telemetry.io), or open an issue to get a conversation started. 
 
 ### Author a Plugin
 The power of Snap comes from its open architecture. Add to the ecosystem by building your own plugins to collect, process or publish telemetry.
 
 * The definitive how-to is in [PLUGIN_AUTHORING.md](docs/PLUGIN_AUTHORING.md)
 * Recommendations to make effective, well-designed plugins are in [PLUGIN_BEST_PRACTICES.md](docs/PLUGIN_BEST_PRACTICES.md)
 
+### Become a Maintainer
+Snap maintainers are here to help guide Snap, the plugins, and the community forward in a positive direction. Think you're ready to be a maintainer? Check out [Responsibilities of a Maintainer](docs/MAINTAINERS.md#responsibilities-of-maintainers) and open an issue requesting to be added to the [Maintainers list](docs/MAINTAINERS.md). 
```

Comment:
I don't want to give the impression that anyone and everyone should get ready to become a maintainer. We do want others to know we're interested in maintainers speaking up though. A rewording here would be good.